ansible/vars/epoch/main.yml

```diff
@@ -0,0 +1,44 @@
+api_base_uri: http://{{ public_ipv4 }}:{{ epoch_config.http.external.port }}/v2
+configure_peers: false
+seed_peers:
+  - "35.177.201.208"
+  - "18.136.116.76"
+  - "34.209.181.77"
+
+epoch_config:
+  peers:
+    - "aenode://pp_2KWhoNRdythXAmgCbM6QxFo95WM4XXGq2pjcbKitXFpUHnPQc3@35.177.201.208:3015"
+    - "aenode://pp_RKVZjm7UKPLGvyKWqVZN1pXN6CTCxfmYz2HkNL2xiAhLVd2ho@18.136.116.76:3015"
+    - "aenode://pp_2gPZjuPnJnTVEbrB9Qgv7f4MdhM4Jh6PD22mB2iBA1g7FRvHTk@34.209.181.77:3015"
+
+  sync:
+    port: 3015
+
+  http:
+    external:
+      port: 3013
+    internal:
+      port: 3113
+
+  keys:
+    dir: keys
+    peer_password: secret
+
+  chain:
+    persist: true
+    db_path: "./db{{ db_version|mandatory }}"
+
+  mining:
+    autostart: false
+    beneficiary: "ak_tjnw1KcmnwfqXvhtGa9GRjanbHM3t6PmEWEWtNMM3ouvNKRu5"
```

Comment:
I see this beneficiary value is the same as UAT.

I see this beneficiary value is not one that clearly has no private key associated with it:
```
(epoch@localhost)4> rp(aehttp_api_encoder:decode(<<"ak_tjnw1KcmnwfqXvhtGa9GRjanbHM3t6PmEWEWtNMM3ouvNKRu5">>)).
{account_pubkey,<<117,122,27,179,167,250,90,199,7,205,
                  232,105,4,39,250,138,32,11,214,144,62,
                  175,120,145,181,47,40,233,15,109,204,
                  128>>}
ok
```

I see that autostart is false so no mining.

Though - just in case anything happens that triggers mining - shall beneficiary be a value that clearly has no corresponding private key?

Reply:
Well, epoch requires `beneficiary` regardless of mining on/off. I'd prefer to make this conditional so we don't have to such `beneficiary` addresses our of our fingers.